plugins/org.eclipse.n4js.tester.ui/src/org/eclipse/n4js/tester/ui/TesterUiActivator.java

```diff
@@ -90,11 +90,10 @@ public void start(final BundleContext context) throws Exception {
 		super.start(context);
 		plugin = this;
 
-		final Injector parentInjector = N4JSActivator.getInstance().getInjector(
-				N4JSActivator.ORG_ECLIPSE_N4JS_N4JS);
-		injector = parentInjector.createChildInjector(Modules.override(new TesterModule()).with(
-				new N4TesterUiModule()));
-
+		Injector parentInjector = N4JSActivator.getInstance().getInjector(N4JSActivator.ORG_ECLIPSE_N4JS_N4JS);
+		TesterModule testerModule = new TesterModule(parentInjector);
+		TesterUiModule testerUiModule = new TesterUiModule(parentInjector);
+		injector = Guice.createInjector(testerModule, testerUiModule);
```

Comment:
I think something similar to `org.eclipse.xtext.ui.shared.internal.SharedStateContributionRegistryImpl.createInjectorWithoutJITBindings(Module)`  would be needed here.